### PR TITLE
Renamed otel-metrics-service folder and updated values file for how to use Grafana Agent with OTEL

### PR DESCRIPTION
**Pull request by magsther**
- Renamed the otel-metrics-service folder to Otel-service
- Updated values.yaml file
- Updated README.md

### CODE REVIEW

File: examples/cluster-with-pvc/output.yaml [hunk 1]
```diff
@@ -44296,6 +44296,8 @@ metadata:
     app.kubernetes.io/name: prometheus-node-exporter
     app.kubernetes.io/instance: k8smon
     app.kubernetes.io/version: "1.7.0"
+  annotations:
+    prometheus.io/scrape: "true"
```
[the remark this answers is not in the document]
Comment (petewall) [reply examples/cluster-with-pvc/output.yaml:44300]: They all appear to be on the Node Exporter Service.
The template for that:
```
{{- if .Values.service.enabled }}
apiVersion: v1
kind: Service
metadata:
  name: {{ include "prometheus-node-exporter.fullname" . }}
  namespace: {{ include "prometheus-node-exporter.namespace" . }}
  labels:
    {{- include "prometheus-node-exporter.labels" $ | nindent 4 }}
  {{- with .Values.service.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
spec:
{{- if .Values.service.ipDualStack.enabled }}
  ipFamilies: {{ toYaml .Values.service.ipDualStack.ipFamilies | nindent 4 }}
  ipFamilyPolicy: {{ .Values.service.ipDualStack.ipFamilyPolicy }}
{{- end }}
  type: {{ .Values.service.type }}
  ports:
    - port: {{ .Values.service.port }}
      {{- if ( and (eq .Values.service.type "NodePort" ) (not (empty .Values.service.nodePort)) ) }}
      nodePort: {{ .Values.service.nodePort }}
      {{- end }}
      targetPort: {{ .Values.service.targetPort }}
      protocol: TCP
      name: {{ .Values.service.portName }}
  selector:
    {{- include "prometheus-node-exporter.selectorLabels" . | nindent 4 }}
{{- end }}
```

Annotations are set simply with:
```
  {{- with .Values.service.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

[the remark this answers is not in the document]
Comment (petewall) [reply examples/cluster-with-pvc/output.yaml:44300]: Ah, sure enough, it's just set in the NE values file:
```yaml
service:
  enabled: true
...
  annotations:
    prometheus.io/scrape: "true"
```

And we try to "unset" it in our values file:
```
prometheus-node-exporter:
...
  # @ignored - Disable prometheus.io/scrape annotation
  service:
    annotations:
      prometheus.io/scrape: null
```

[the remark this answers is not in the document]
Comment (petewall) [reply examples/cluster-with-pvc/output.yaml:44300]: What version of the Helm cli are you using? Maybe that's why it's not cascading this value downwards.
FWIW, I'm running:
```bash
% helm version
version.BuildInfo{Version:"v3.14.2", GitCommit:"c309b6f0ff63856811846ce18f3bdc93d2b4d54b", GitTreeState:"clean", GoVersion:"go1.22.0"}
```

[the remark this answers is not in the document]
Comment (magsther) [reply examples/cluster-with-pvc/output.yaml:44300]: @petewall It's unclear why the other files were changes, I only made changes to the otel-service folder. After that I ran `make regenerate-example-outputs` and then excluded the other folders from the commit.

File: examples/otel-service/values.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    grpc:
+      enabled: true
+    thriftBinary:
+      enabled: true
+    thriftCompact:
+      enabled: true
+    thriftHttp:
+      enabled: true
+  zipkin:
+    enabled: true
```
[the remark this answers is not in the document]
Comment (magsther) [reply examples/otel-service/values.yaml:52]: We don't need them. I just thought it made the example more clear to add the `otlp` receivers.

[the remark this answers is not in the document]
Comment (magsther) [reply examples/otel-service/values.yaml:52]: Hi @petewall just checking if there is anything I still need to do?

[the remark this answers is not in the document]
Comment (petewall) [reply examples/otel-service/values.yaml:52]: Rebase your fork with the latest from main. Then run `make regenerate-example-outputs` again.

Have you checked what your Helm cli version is? Can you report that that might be.

**Comment by petewall** [comment]
I just had someone else with the same annotation issue. What version of the Helm CLI are you using? Try upgrading to the latest and trying again?

**Comment by magsther** [comment]
> I just had someone else with the same annotation issue. What version of the Helm CLI are you using? Try upgrading to the latest and trying again?

`version.BuildInfo{Version:"v3.9.0", GitCommit:"7ceeda6c585217a19a1131663d8cd1f7d641b2a7", GitTreeState:"clean", GoVersion:"go1.18.2"}`

**Comment by petewall** [comment]
> > I just had someone else with the same annotation issue. What version of the Helm CLI are you using? Try upgrading to the latest and trying again?
> 
> `version.BuildInfo{Version:"v3.9.0", GitCommit:"7ceeda6c585217a19a1131663d8cd1f7d641b2a7", GitTreeState:"clean", GoVersion:"go1.18.2"}`

Gotcha! Yeah, there's an issue in helm 3.9 where it doesn't respect removing fields in values by setting them to null. That's why the node exporter annotation keeps coming back. Try upgrading your helm cli and try again. I'm really looking forward to getting this example updated!

**Comment by magsther** [comment]
> > > I just had someone else with the same annotation issue. What version of the Helm CLI are you using? Try upgrading to the latest and trying again?
> > 
> > 
> > `version.BuildInfo{Version:"v3.9.0", GitCommit:"7ceeda6c585217a19a1131663d8cd1f7d641b2a7", GitTreeState:"clean", GoVersion:"go1.18.2"}`
> 
> Gotcha! Yeah, there's an issue in helm 3.9 where it doesn't respect removing fields in values by setting them to null. That's why the node exporter annotation keeps coming back. Try upgrading your helm cli and try again. I'm really looking forward to getting this example updated!

My system got a bit messed up, I think it would be easier for me to close this PR and create a new one.

**Pull request closed without merging**